### PR TITLE
Hide the customized default setting by default in Rancher chart

### PR DESCRIPTION
**Pull request by meldafrawi**
https://github.com/longhorn/longhorn/issues/1308

Signed-off-by: Mohamed Eldafrawi <mohamed.eldafrawi@rancher.com>

 Longhorn Default Settings: False
image

 Longhorn Default Settings: True
image

### CODE REVIEW

File: chart/questions.yml [hunk 1]
```diff
@@ -100,133 +100,141 @@ questions:
...
-  type: int
-  min: 0
-  max: 100
-  default: 25
-- variable: defaultSettings.upgradeChecker
-  label: Enable Upgrade Checker
-  description: 'Upgrade Checker will check for new Longhorn version periodically. When there is a new version available, it will notify the user using UI. By default true.'
-  group: "Longhorn Default Settings"
+  description: "Customize Longhorn First Time Deploy Default Settings (will only work if ther is no Longhorn installation exist on the cluster)"
+  label: "Longhorn Default Settings"
```
Comment (yasker) [inline chart/questions.yml:106]: `Customize Default Setting`

Comment (meldafrawi) [reply chart/questions.yml]: fixed

File: chart/questions.yml [hunk 2]
```diff
@@ -100,133 +100,141 @@ questions:
...
-  group: "Longhorn Default Settings"
-  type: int
-  min: 0
-  max: 100
-  default: 25
-- variable: defaultSettings.upgradeChecker
-  label: Enable Upgrade Checker
-  description: 'Upgrade Checker will check for new Longhorn version periodically. When there is a new version available, it will notify the user using UI. By default true.'
-  group: "Longhorn Default Settings"
+  description: "Customize Longhorn First Time Deploy Default Settings (will only work if ther is no Longhorn installation exist on the cluster)"
```
Comment (yasker) [inline chart/questions.yml:105]: Typo on `ther`.

And don't use `CamelCase` when writing a sentence. `Customize the default settings before installing Longhorn for the first time. This option will only work if the cluster hasn't installed Longhorn.`

Comment (meldafrawi) [reply chart/questions.yml]: fixed

**Pull request merged**